### DISCUSSION TITLE
Can we get `temp_step` to be configurable?

### DISCUSSION THREAD

**Discussion by TheBestMoshe** [revk/xESP32-Faikout#510] [Q&A] [answered]
When using S21 it is hard-coded to `0.5`. This is sent to Home Assistant. I'd rather use `1`.

I believe [ it's possible to customize it](https://www.home-assistant.io/docs/configuration/customizing-devices/#customizing-an-entity-in-yaml), but I'd rather not have customizations floating around if I can configure it.

https://github.com/revk/ESP32-Faikin/blob/909e776f6d75a20161e43f3d673d5f4b5f58669b/ESP/main/Faikin.c#L211

**Comment by MuzzaMcD** [comment, marked as answer]
Under Settings there is an option called 'ha1c' . 'This will force 1C steps in HA temp setting'

**Reply by TheBestMoshe** [reply]
Thanks. This worked for Fahrenheit too.
